packages/shared/src/components/modals/BootPopups.tsx

```diff
@@ -102,7 +102,7 @@ export const BootPopups = (): ReactElement => {
    * Boot popup based on marketing CTA
    */
   useEffect(() => {
-    if (marketingCtaPlus && isValidRegion && !user?.isPlus) {
+    if (marketingCtaPlus && !isIOSNative() && isValidRegion && !user?.isPlus) {
```

Comment:
cc: @idoshamun I was mentioning probably should not show on iOS because it is not supported there officially, could get us into issues with Apple (same as plus page opening iOS app)

Reply:
No problem, but @omBratteng should provide support for it once he's ready

Reply:
AS-1030